tests/kafkatest/services/console_consumer.py

```diff
@@ -158,6 +165,14 @@ def _worker(self, idx, node):
         else:
             prop_file = self.render('console_consumer.properties')
 
+        # Add security properties to the config. If security protocol is not specified,
+        # use the default in the template properties.
+        security_config = SecurityConfig(node.account, self.security_protocol, prop_file)
+        prop_file += str(security_config)
+        if self.security_protocol is None:
+            self.security_protocol = security_config.security_protocol
+            if self.security_protocol == SecurityConfig.SSL:
+                self.new_consumer = True
```

Comment:
Agreed that we should throw an exception if an invalid combination is specified as some people may not be aware that SSL is not supported in the old consumer.